config/hirte/README-hirte

```diff
@@ -0,0 +1,3 @@
+This directory is for user costume configuration for hirte.
```

Comment:
Please rename the file to `README.md`, put it under `config/hirte/hirte.conf.d` subdirectory and use similar content as for the agent